packages/fxa-content-server/app/scripts/views/ready.js

```diff
@@ -198,4 +198,9 @@ Cocktail.mixin(
   PocketMigrationMixin
 );
 
+export const legacyL10nStrings = [
+  t('You are now ready to make changes to your Firefox Account.'),
+  t('Firefox Accounts can only be placed into an IFRAME on approved sites'),
```

Comment:
`Firefox Accounts can only be placed into an IFRAME on approved sites`was not in this ready.js file, and we likely only need the legacy string once (in auth-errors)

Reply:
Fixed